runhouse/resources/hardware/sagemaker/sagemaker_cluster.py

```diff
@@ -1241,9 +1240,7 @@ def _sync_runhouse_to_cluster(self, node: str = None, _install_url=None, env=Non
         local_rh_package_path = Path(pkgutil.get_loader("runhouse").path).parent
 
         # **Note** temp patch to handle PyYAML errors: https://github.com/yaml/pyyaml/issues/724
-        base_rh_install_cmd = (
-            f'{self._env_activate_cmd} && python3 -m pip install "cython<3.0.0"'
-        )
+        base_rh_install_cmd = f'{self._get_env_activate_cmd(env=None)} && python3 -m pip install "cython<3.0.0"'
```

Comment:
@jlewitt1 for sagemaker, should this be using the env that's passed in or just defaulting to None and installing in the base default env?

Reply:
this should work since right now we only use the default env